client/components/Editor/styles/main.scss

```diff
@@ -67,3 +67,31 @@ sub {
 sup {
 	vertical-align: super;
 }
+
+/*
+ * Replace the list markers with equivalent ::before pseudo elements. This allows us to style them more
+ * freely than the ::marker pseudo element, which doesn't support properties like border and background
+ * that we use to indicate suggested edits. 
+ */
+
+li {
+	list-style-type: none;
+	position: relative;
+}
+
+li::before {
+	margin-left: -1em;
+	position: absolute;
+}
+
+ul > li::before {
+	content: '•';
```

Comment:
This isn't really a solution, since it breaks our nested unordered lists
pub pub now:
<img width="439" alt="image" src="https://github.com/pubpub/pubpub/assets/473542/6eac5715-76a2-4cef-8bc3-57a308a2e577">

pubpub after this change:
<img width="332" alt="image" src="https://github.com/pubpub/pubpub/assets/473542/92965ee7-8b9c-45d2-bdf0-07a31853a6e7">

Reply:
Does something like this work (not sure if the characters are right)?
```scss
ul > li::before {
  content: "•";
}

ul > li > ul > li::before {
  content: "◦";
}

ul > li > ul > li > ul li::before {
  content: "■";
}
```

Reply:
I wrote a terrible sass function that would let us target list items of arbitrary depth.
```scss
@use 'sass:selector';

@function at-depth($child, $depth: 0, $parent: '*', $extend: false) {
	$result: '#{$parent} > #{$child}';
	@for $i from 0 through $depth {
		@if $i > 0 {
			$result: $result +
				selector.nest(
					'> #{$parent}',
					"#{if($i == $depth and $extend == true, '', '>')}#{$child}"
				);
		}
	}
	@return $result;
}

#{at-depth('li', 0, 'ul')}::marker { content: '•'; }
#{at-depth('li', 1, 'ul')}::marker { content: '◦'; }
#{at-depth('li', 2, 'ul', true)}::marker { content: '■'; }
```